.storybook/DocumentationTemplate.mdx

```diff
@@ -18,12 +18,10 @@ import {
 <Subtitle />
 <Description />
 
-<Primary />
+<Stories title="Variants" />
 
 ## Properties
 
 The component accepts the following inputs (properties):
 
 <Controls />
-
-<Stories includePrimary={false} />
```

Comment:
It aligns more with the .mdx Docs pages if all the stories are together before the controls.